src/celeritas/ext/RootInterfaceClasses.hh

```diff
@@ -0,0 +1,7 @@
+#include "io/ImportData.hh"
```

Comment:
genreflex has a "feature" where it embeds the argument directly in the source files.  Beside the huge duplication if we pass the actual headers, in case of Celeritas if flat out fails because of the `pragma once` which are rejected in that context.

Reply:
🪄🦄🧙

Reply:
![image](https://user-images.githubusercontent.com/5175087/214156661-681e4834-ac4a-4ae4-9688-c1fcaa2ccd0e.png)